### DISCUSSION TITLE
Change Servers Order by drag n drop

### DISCUSSION THREAD

**Discussion by benjamin74** [spatie/ray#616] [Ideas]
Hello,

Unfortunately I created my Servers in an order that don't make much sense...

It would be nice to have a way to reorder them by drag n drop.

Or, if you implement an export / import function and if the export format is readable, I could re-arrange them using Sublim text and doing an export / delete all servers / edit order in sublime / import newly ordered servers.

Thanks,

**Comment by AdrianMrn** [comment]
Good idea. I'll add this feature request to our internal issue tracker. Compared to the other things we're working on right now, this will be somewhat low-priority, so I can't promise when I'll add it yet!

See my answer in your other thread for how to export/import, you could probably change the order of the servers there (make sure to do it while Ray is turned off).
